src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,40 @@
+package duke.task;
+import duke.exceptions.*;
```

Comment:
Imported classes should always be listed explicitly.
```suggestion
import duke.exceptions.ParsingTaskException;
```